internal/reader/adapter_v1.go

```diff
@@ -266,16 +266,23 @@ func (a AdapterV1) GetInputs(
 		}
 		if where.Type != nil {
 			inputboxField := "InputBoxIndex"
-			defaultStr := "-1"
+			espressoStr := "-1"
```

Comment:
Why not the "type" field?

Reply:
cc @ZzzzHui

Reply:
because we can already differentiate the type based on their "InputBoxIndex" field, right? What do you think.

Reply:
I'd rather go with the type field.. it is more clear. The input box field we'll have to map each value to its sequencer/DA in the code ..

Reply:
yes for multiple sequencers, it's more clear to have a type field

Reply:
> Why not the "type" field?

I thought the question was for me. But it's actually for Diego lol